evap/staff/importers/enrollment.py

```diff
@@ -145,9 +145,9 @@ def __init__(self, *args, invalid_is_graded: str, **kwargs):
     @classmethod
     def is_graded_from_import_string(cls, is_graded: str) -> bool:
         is_graded = is_graded.strip()
-        if is_graded == settings.IMPORTER_GRADED_YES:
+        if is_graded in settings.IMPORTER_GRADED_YES:
             return True
-        if is_graded == settings.IMPORTER_GRADED_NO:
+        if is_graded in settings.IMPORTER_GRADED_NO:
```

Comment:
I believe there should be a test for, probably testing with "yes" and "no" and some invalid value. Maybe use any of the non-first options in there (so we have that covered)?